src/fable-fsharp/Replacements/Replacements.fs

```diff
@@ -1092,6 +1092,9 @@ module private CoreLibPass =
             fsharp + "Collections.Seq" => ("Seq", Static)
             fsharp + "Collections.Set" => ("Set", Static)
             fsharp + "Core.Choice" => ("Choice", Both)
+            fsharp + "Control.Event" => ("Event", Both)
```

Comment:
Not entirely sure and actually it won't make any real difference in this case, but I think we can use `Static` here.